tests/integration/popular_collections/POCLOUD.txt

```diff
@@ -0,0 +1,100 @@
+C2799438299-POCLOUD
+C1996881146-POCLOUD
+# C2204129664-POCLOUD
```

Comment:
This collection isn't working so good :face_with_thermometer: 

We get 0 granules:

```
>           assert len(granules) > 0, msg
E           AssertionError: AssertionError for C2204129664-POCLOUD
E           assert 0 > 0
E            +  where 0 = len([])
```